lyvely-angular-code-review/src/app/app.component.ts

```diff
@@ -0,0 +1,24 @@
+import {Component} from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  title = 'lyvely-angular-code-review';
+
+  changeColour() {
+    const b = document.getElementById('background-sky');
```

Comment:
We should avoid direct dom manipulation. We can use the same 'isNight' we discussed above to conditionally add the correct classes using ngClass directive. And use the function 'changeColour' (which we can rename to something that signifies this toggling) to toggle its value between true and false